src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -147,6 +147,12 @@ public class DurableTaskExtension :
         /// </value>
         public bool DisableHttpManagementApis { get; set; }
 
+        /// <summary>
+        /// Enables or disables logging to Application Insights.
+        /// </summary>
+        /// <value>True to enable, false to disable. The default value is <c>true</c>.</value>
+        public bool LoggingEnabled { get; set; } = true;
```

Comment:
@christopheranderson earlier suggested we should use explicit "enabled" and "disabled" text flags to toggle app insights logging - i.e. this should be an enum. That would allow us to add more fine-grained logging filters later.

Reply:
I thought we agreed on bool.  To avoid additional churn, could you propose the shape you want this to be in?

---
In reply to: [137172410](https://github.com/Azure/azure-functions-durable-extension/pull/37#discussion_r137172410) [](ancestors = 137172410)

Reply:
I don't think you should need this at all as log filtering should be controlled externally from triggers/bindings.